genesyscloud/provider/provider.go

```diff
@@ -433,14 +306,16 @@ func InitClientConfig(data *schema.ResourceData, version string, config *platfor
 		})
 	}
 
-	log.Printf("Initialized Go SDK Client. Debug=%t", data.Get("sdk_debug").(bool))
+	if isDefaultConfig {
+		log.Printf("Initialized Go SDK Client. Debug=%t", data.Get("sdk_debug").(bool))
+	}
```

Comment:
I don't really understand this conditional logging?

Reply:
Otherwise this log will be showing up every single time a new client is created. It blows up the logging, especially in test output. This conditional only shows a single log.

Reply:
I'll add a comment